external/autopygmentize

```diff
@@ -14,8 +14,13 @@ options=${@:1:$(($#-1))}  # handle others args as options to pass to pygmentize
 file_common_opts="--brief --dereference"
 
 lexer=$(pygmentize -N "$file")
+if [[ "$lexer" == resource ]]; then
+    # Resource lexer grabs "*.txt", but almost always that's plain text
```

Comment:
I've removed that association now, this if can be removed.

Reply:
I've removed this `if`; thanks!